frontend/src/views/omni/Extensions/ExtensionsPicker.vue

```diff
@@ -5,7 +5,7 @@ Use of this software is governed by the Business Source License
 included in the LICENSE file.
 -->
 <template>
-  <div class="flex flex-col gap-2 overflow-hidden">
+  <div class="flex flex-col gap-2 overflow-hidden min-h-36">
```

Comment:
Everywhere else in the codebase minimal height is defined in css styles and in pixels. Is there any reason for why that is? Please let me know if this should be changed to match other definitions.

Reply:
What you did here is correct. Pixels is more of a legacy code which we need to fix some day.